### DISCUSSION TITLE
No network connection on mac guest when running VPN on host

### DISCUSSION THREAD

**Discussion by delro92** [cirruslabs/tart#727] [General]
I'm getting no network connection when running Mullvad on the host m1 Macbook. I tried running with the "--net-softnet" parameter but still no luck. Any recommendations? Thanks!

**Comment by edigaryev** [comment]
Consider using a VPN solution with [Network Extension](https://developer.apple.com/documentation/networkextension) support, for example, [WireGuard](https://www.wireguard.com/install/), for connecting to your VPN.

See https://github.com/cirruslabs/tart/issues/473#issuecomment-1516580781 for more details.

**Reply by delro92** [reply]
Woah, thank you so much for the quick reply, I'll look into it right away, THANKS!
